backend/clubs/serializers.py

```diff
@@ -998,6 +999,26 @@ class Meta:
         ]
 
 
+class UserClubVisitSerializer(serializers.ModelSerializer):
+    """
+    Used by users to get a list of clubs that they have visited.
+    """
+
+    person = serializers.HiddenField(default=serializers.CurrentUserDefault())
+    club = ClubListSerializer(read_only=True)
+
+    class Meta:
+        model = ClubVisit
+        fields = ("club", "person")
+
+
+class UserClubVisitWriteSerializer(UserSubscribeSerializer):
+    club = serializers.SlugRelatedField(queryset=Club.objects.all(), slug_field="code")
+
+    class Meta(UserClubVisitSerializer.Meta):
```

Comment:
I'm pretty sure you can exclude the Meta class and it'll pick up the super one. Not 100% sure though, but it works when I tried it with events.